File: CS/Q147080/Tab/ViewInfo/MyWindowsXPTabHeaderViewInfo.cs
﻿using DevExpress.XtraTab.ViewInfo;

namespace DxSample.Tab.ViewInfo {
    public class MyWindowsXPTabHeaderViewInfo : WindowsXPTabHeaderViewInfo {
        public MyWindowsXPTabHeaderViewInfo(BaseTabControlViewInfo viewInfo) : base(viewInfo) { }
        private MyTabRowViewInfoCollection rows = new MyTabRowViewInfoCollection();
        public override BaseTabRowViewInfoCollection Rows { get { return rows; } }
        protected override bool UpdateFillPageBounds() { return false; }
    }
}

File: VB/Q147080/Tab/ViewInfo/MyWindowsXPTabHeaderViewInfo.vb
Imports DevExpress.XtraTab.ViewInfo

Namespace DxSample.Tab.ViewInfo
    Public Class MyWindowsXPTabHeaderViewInfo
        Inherits WindowsXPTabHeaderViewInfo

        Public Sub New(ByVal viewInfo As BaseTabControlViewInfo)
            MyBase.New(viewInfo)
        End Sub

        Private rows_Renamed As New MyTabRowViewInfoCollection()
        Public Overrides ReadOnly Property Rows() As BaseTabRowViewInfoCollection
            Get
                Return rows_Renamed
            End Get
        End Property
        Protected Overrides Function UpdateFillPageBounds() As Boolean
            Return False
        End Function
    End Class
End Namespace
